internal/provider/workflow_job_resource_test.go

```diff
@@ -339,6 +368,38 @@ func TestAccAAPWorkflowJob_UpdateWithNewInventoryIdPromptOnLaunch(t *testing.T)
 				Check: resource.ComposeAggregateTestCheckFunc(
 					resource.TestMatchResourceAttr("aap_workflow_job.test", "status", regexp.MustCompile("^(failed|pending|running|complete|successful|waiting)$")),
 					resource.TestMatchResourceAttr("aap_workflow_job.test", "url", regexp.MustCompile("^/api(/controller)?/v2/workflow_jobs/[0-9]*/$")),
+
+					testAccCheckWorkflowJobUpdate(&jobURLBefore, true),
+					// Wait for the job to finish so the inventory can be deleted
+					testAccCheckWorkflowJobPause(ctx, "aap_workflow_job.test"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAAPWorkflowJob_UpdateWithNewInventoryIdPromptOnLaunch2(t *testing.T) {
+	// In order to run the this test for the workflow job resource, you must have a working job template already in your AAP instance.
+	// The job template used must be set to require an inventory on launch. Export the id of this job template into the
+	// environment variable AAP_TEST_WORKFLOW_JOB_TEMPLATE_ID. Otherwise this test will fail when running the suite.
+
+	var jobURLBefore string
+
+	inventoryName := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	jobTemplateID := os.Getenv("AAP_TEST_WORKFLOW_JOB_TEMPLATE_ID")
+	ctx := context.Background()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccWorkflowJobResourcePreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read testing
+			{
+				Config: testAccUpdateWorkflowJobWithInventoryID(inventoryName, jobTemplateID),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestMatchResourceAttr("aap_workflow_job.test", "status", regexp.MustCompile("^(failed|pending|running|complete|successful|waiting)$")),
+					resource.TestMatchResourceAttr("aap_workflow_job.test", "url", regexp.MustCompile("^/api(/controller)?/v2/workflow_jobs/[0-9]*/$")),
+					resource.TestCheckResourceAttrPair("aap_workflow_job.test", "inventory_id", "aap_inventory.test", "id"),
```

Comment:
Is this a duplicate? this looks like a copy of an existing test (with a `2`) that maybe was a starting point to rework, but maybe that didn't happen?